examples/common/vtorc/config.yaml

```diff
@@ -0,0 +1 @@
+instance-poll-time: 1s
```

Comment:
It will be good to pad this out with other commonly used properties so that we can point people to it as an example. I'd suggest leaving out the audit flags, but everything else seems useful.